operations/helm/charts/mimir-distributed/templates/validate.yaml

```
@@ -58,10 +58,12 @@
         (ne .memcached.addresses (include "mimir.chunksCacheAddress" $))
         }}
       {{- fail (cat 
-          "chunks cache is enabled, but blocks_storage.bucket_store.chunks_cache.memcached.addresses is not set to the correct address, expected " 
-          (include "mimir.chunksCacheAddress" $) 
-          ", but got" 
-          .memcached.addresses) 
+          "you set chunks-cache.enabled, but the address of the chunks cache in the calculated Mimir configuration in 'blocks_storage.bucket_store.chunks_cache.memcached.addresses'"
```

calculated is an internal term

```suggestion
          "you set chunks-cache.enabled, but the address of the chunks cache in the Mimir configuration in 'blocks_storage.bucket_store.chunks_cache.memcached.addresses'"
```